engine/docker/docker.go

```diff
@@ -347,15 +347,15 @@ func (e *Docker) tail(ctx context.Context, id string, output io.Writer) error {
 
 	logs, err := e.client.ContainerLogs(ctx, id, opts)
 	if err != nil {
+		logger.FromContext(ctx).WithError(err).
+			WithField("container", id).
+			Debugln("failed to stream logs")
 		return err
 	}
+	defer logs.Close()
+
+	stdcopy.StdCopy(output, output, logs)
```

Comment:
log the error in case stdcopy returns an err

Reply:
Done